src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_panel.scss

```diff
@@ -52,3 +52,8 @@
   // keeps outline from getting cut off by other panels without margins
   z-index: 999 !important;
 }
+
+// allows scrolling up on lens visualiztions in edit mode
+.coreSystemRootDomElement {
+  overflow-y: auto !important;
+}
```

Comment:
I believe allowing y-overflow leads to some strange behaviour when trying to scroll when both the editor + dashboard have scrollbars:


https://github.com/user-attachments/assets/e080454f-9e18-49ee-b2ee-d201aa5798aa

I will default to @cqliu1's opinion here, but I think the best solution here is to just change the scroll into view logical position method? 🤔 And `nearest` seems to work well IMO

Reply:
fyi I stumbled across the [`scroll-margin` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-margin) - and it seems like it will be a good helper here :+1: